WTWSMS/history/characters/levantine_arabic.txt

```diff
@@ -58,7 +58,7 @@
 	intrigue=5
 	stewardship=5
 	religion="nicene"
-	culture="levantine_arabic"
+	culture="aramean"
```

Comment:
Did you use "replace all" in this file or was every character checked separately?

Reply:
> 
> 
> Did you use "replace all" in this file or was every character checked separately?

Every character was checked separately. Don't bother yourself reviewing of this file, Loup already doing it in previous PR.

Reply:
OK, to be honest I'm relieved 😀